<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortShowWidget</class>
 <widget class="QWidget" name="SerialPortShowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialPortShowWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView_recv"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_clear">
       <property name="text">
        <string>清空内容</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_recv_start">
       <property name="text">
        <string>开始接收</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_recv_stop">
       <property name="text">
        <string>停止接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
